### PR TITLE
Add a small border to the description dot for easier mouse-over

### PR DESCRIPTION
**Pull request by DanielBiegler**
## Scope

What's changed:

- Added a small border to the description dot for easier mouse-over
- Used variable for bg color and made it not too big, only 6px

## Potential Risks / Drawbacks

- None. Small css fix.

## Review Questions

- You like it? I like it, way easier to hover now.

| Light | Dark |
|--------|--------|
| image | image | 

---

Was mentioned in #19952 for a standalone PR.

### CODE REVIEW

**Review by rijkvanzanten — APPROVED** [review]
Love it. Great tweak!

File: app/src/components/v-table/table-header.vue [hunk 1]
```diff
@@ -441,6 +441,8 @@ function toggleManualSort() {
 	background-color: var(--foreground-subdued);
 	display: inline-block;
 	border-radius: 50%;
+	border: var(--background-page) 6px solid;
+	box-sizing: content-box;
```
Comment (rijkvanzanten) [inline app/src/components/v-table/table-header.vue:445]: Here's a great way to test the CSS skill of future maintainers lol

File: app/src/components/v-table/table-header.vue [hunk 2]
```diff
@@ -441,6 +441,8 @@ function toggleManualSort() {
 	background-color: var(--foreground-subdued);
 	display: inline-block;
 	border-radius: 50%;
+	border: var(--background-page) 6px solid;
```
Comment (paescuj) [inline app/src/components/v-table/table-header.vue:444]: Would be a bit cleaner IMO
```suggestion
	border: transparent 6px solid;
```

Comment (DanielBiegler) [reply app/src/components/v-table/table-header.vue:444]: No that doesnt work because then you'll see the background color underneath :angel:

Comment (paescuj) [reply app/src/components/v-table/table-header.vue:444]: Of course, I didn't notice the dot is composed out of background-color / radius 😅👍

**Pull request merged**